ApiSamples/Base.lproj/Localizable.strings

```diff
@@ -201,3 +201,27 @@
 
 /* No comment provided by engineer. */
 "SampleDescription_CancelSearch" = "Cancel an ongoing search before it completes";
+
+/* No comment provided by engineer. */
+"SampleTitle_FindOutdoorRoute" = "Find an outdoor route";
+
+/* No comment provided by engineer. */
+"SampleDescription_FindOutdoorRoute" = "Find a route between two outdoor locations";
```

Comment:
The 3 route example descriptions here don't match the Android description strings (https://github.com/wrld3d/android-api-samples/blob/master/apisamples/src/main/res/values/strings.xml) - believe we want these to be equivalent where possible to help bridge between the two platform APIs.  The Titles and the CancelRoutingQuery sample strings do match.

Reply:
Descriptions updated.
https://github.com/wrld3d/ios-api-example/pull/6/commits/f86e1269521d067b8ec18235ad10808dd410841c